Userland/Libraries/LibWeb/CSS/StyleProperties.cpp

```diff
@@ -355,6 +355,23 @@ Optional<CSS::TextAlign> StyleProperties::text_align() const
     }
 }
 
+Optional<CSS::PointerEvents> StyleProperties::pointer_events() const
+{
+    auto value = property(CSS::PropertyID::PointerEvents);
+    if (!value.has_value() || !value.value()->is_identifier())
+        return {};
+
+    switch (static_cast<const IdentifierStyleValue&>(*value.value()).id()) {
+    case CSS::ValueID::Auto:
+        return CSS::PointerEvents::Auto;
+    case CSS::ValueID::None:
+        return CSS::PointerEvents::None;
+    // TODO: Implement Initial and Inherit
```

Comment:
`initial` and `inherit` are handled automatically, so they don't need to be dealt with here - the TODO can be removed.